lib/python/machinekit/service.py

```diff
@@ -89,6 +90,24 @@ class Service(object):
     A simple class to publish a Machinekit network service using zeroconf.
     """
 
+    # See https://gist.github.com/bgusach/a967e0587d6e01e889fd1d776c5f3729
+    def __multi_replace(self, string, replacements, ignore_case=False):
```

Comment:
I think there should be `_` rather than a `__`

Reply:
I can change that. I'm new to python and I saw that convention for a private type in a class that external developers should avoid using is the double underscore. If you have been using single underscore, I will conform to the existing pattern.